gdc_client/exceptions.py

```diff
@@ -1,3 +1,6 @@
 class ClientError(Exception):
-    """ Base client error.
-    """
+    """Base client error."""
+
+
+class HTTPError(Exception):
+    """Base HTTP error."""
```

Comment:
We should prob capture the status code of the response for this as a property of this exception. And I would suggest making this an exception vs an error.